packages/cli-lib/lib/watch.js

```diff
@@ -65,7 +65,8 @@ async function uploadFile(accountId, file, dest, options) {
     // Because we cannot clear ES Modules from the cache, we need to load them in a separate process.
     // So this spawns a new node process that calls `hs upload` with the same options as watch was called with.
     // More context: https://github.com/HubSpot/hubspot-cli/pull/712#discussion_r945056954
-    const fieldsJsProcessCode = `require('@hubspot/cli/commands/upload').handler(${parsedOptions})`;
+    const uploadCommandPath = require.resolve('@hubspot/cli/commands/upload');
+    const fieldsJsProcessCode = `require('${uploadCommandPath}').handler(${parsedOptions})`;
```

Comment:
We need to resolve the path from the cli-lib module because if we require it directly in the child process, it expects the CWD to have `node_modules/@hubspot/cli`

Reply:
NOTE: untested on Windows